Dockerfile

```diff
@@ -67,22 +67,18 @@ RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == syste
 ## Remove any existing configurations
 RUN rm -f /etc/httpd/conf.d/*
 
-## For Ruby
-ENV TERM=xterm \
-    LANG=en_US.UTF-8 \
-    RUBY_GEMS_ROOT=/opt/rubies/ruby-2.3.1/lib/ruby/gems/2.3.0 \
-    PATH=$PATH:/opt/rubies/ruby-2.3.1/bin
+## For ruby
+ENV RUBY_GEMS_ROOT=/usr/local/lib/ruby/gems/2.5.0 \
+    LANG=en_US.UTF-8
 
 # Install repos
 RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
-    curl -sL https://copr.fedorainfracloud.org/coprs/postmodern/ruby-install/repo/fedora-25/postmodern-ruby-install-fedora-25.repo -o /etc/yum.repos.d/ruby-install.repo && \
-    sed -i 's/\$releasever/25/g' /etc/yum.repos.d/ruby-install.repo
+    curl -sL https://copr.fedorainfracloud.org/coprs/manageiq/ManageIQ-Master/repo/epel-7/manageiq-ManageIQ-Master-epel-7.repo -o /etc/yum.repos.d/manageiq.repo
 
 # Install ruby-install and make
 RUN yum -y install --setopt=tsflags=nodocs ruby-install make
 
-# Install Ruby 2.3.1
-RUN ruby-install ruby 2.3.1 -- --disable-install-doc && rm -rf /usr/local/src/* && yum clean all
+RUN ruby-install --system ruby 2.5.5 -- --disable-install-doc --enable-shared && rm -rf /usr/local/src/* && yum clean all
```

Comment:
Should this be 2.5.7?

Reply:
I just used what was currently in container-ruby. We can go right to 2.5.7 if you want though.

Reply:
I think we want upstream master to always have the latest release of our supported rubies... we'll always be behind but it's worth getting the latest when we can...

and release branches will narrow down to the single version we're supporting...

Reply:
Does latest here mean moving to 2.6? Or are you still just talking about 2.5.7?

Reply:
Agreed, it's just this repo and some other container image repos only have `master` branch.

Reply:
I think 2.5.7 is the master supported version with 2.6.5 being the one we're slowly moving to.  We make the rules so we can decide how we want to do it.  

But, yes, 2.5.7 seems like the best choice here.